### PR TITLE
Added truststore password input for TLS connection

### PR DESCRIPTION
**Pull request by anilkumargedela**
- Added truststore password input field with truststore path to support secure TLS connection between SRS and Elasticsearch

### CODE REVIEW

**Review by taz-pega-work — APPROVED** [review]
looks better ;)

**Review by kingakowalska1 — APPROVED** [review]
Look great!

File: charts/backingservices/charts/srs/README.md [hunk 1]
```diff
@@ -76,6 +76,8 @@ make external-es-secrets NAMESPACE=pegabackingservices ELASTICSEARCH_VERSION=7.1
...
+| `certificateName`                       | Enter the tls certificate name.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (bhowd1) [inline charts/backingservices/charts/srs/README.md:79]: can we also add the default certificate name which is used if the certificateName value is not provided? (`elastic-certificates.p12`)

**Review by chandra-kalavakuri — APPROVED** [review]
lgtm

**Pull request merged**